extruct/uniform.py

```diff
@@ -23,6 +24,22 @@ def _umicrodata_microformat(extracted, schema_context):
     return res
 
 
+def _udublincore(extracted):
+    out = []
+    for obj in extracted:
+        context = obj.pop('namespaces', None)
+        obj['@context'] = context
+        elements = obj['elements']
+        for element in elements:
+            for key, value in element.items():
+                if get_lower_attrib(value) == 'type':
+                    obj['@type'] = element['content']
+                    elements.remove(element)
```

Comment:
I think it's not recommended to modify the list which is being iterated - see e.g. https://stackoverflow.com/questions/1637807/modifying-list-while-iterating - so here I'd suggest using ``for element in list(elements):`` above, and maybe also to be extra clear, use ``obj['elements'].remove(element)`` here, so that it's clear why this list modification is affecting the output.